src/test/java/javaserver/ResponderTest.java

```diff
@@ -1,40 +1,124 @@
 package javaserver;
 
-import java.io.ByteArrayOutputStream;
-import java.io.DataOutputStream;
+import static org.junit.Assert.*;
+
 import java.io.IOException;
-import java.util.HashMap;
 
 import org.junit.Before;
 import org.junit.Test;
 
 public class ResponderTest {
-	ResponseBuilder responseBuilder;
+	Responder testResponseBuilder;
 	SocketWriter writer;
 	Responder responder;
-	String twoHundred;
-	String threeOhTwo;
-	String fourOhFour;
-	String allowedMethodOptions;
+
+	String codedURI = "/parameters?variable_1=Operators%20%3C%2C%20%3E%2C%20%3D%2C%20!%3D%3B%20%2B%2C%20-%2C%20*%2C%20%26%2C%20%40%2C%20%23%2C%20%24%2C%20%5B%2C%20%5D%3A%20%22is%20that%20all%22%3F&variable_2=stuff";
+
+	Request getRoot = new Request("GET", "/");
```

Comment:
This ties into my comment above with the Request holding too much information about itself. Your tests are coupled to the implementation as you are testing for specific routes that are defined elsewhere.